CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Versions
 
+## 1.0.7
+    *Add NSAdvertisingAttributionReportEndpoint to info.plist file.
```

Comment:
Since this is markdown, let's take advantage of that. Also, formatting (space after the `*`).
```suggestion
    * Automatically add `NSAdvertisingAttributionReportEndpoint` to `Info.plist`.
```